Makefile

```
@@ -33,6 +33,41 @@ down:
+test-py-unit:
+	@echo "Running Python unit tests..."
+	cd apps/ai && pytest tests/ -v --tb=short -k "not e2e"
+
+# Run Python E2E tests
+test-py-e2e:
+	@echo "Running Python E2E tests..."
+	$(MAKE) infra-up
+	cd apps/ai && pytest tests/ -v --tb=short -k "e2e"
+
+# Stop all Docker services
+down:
```

checkmake (0.3.2)

[warning] Target "down" defined multiple times (lines 31 and 66).

(uniquetargets)